minetest.conf.example

```diff
@@ -1468,6 +1468,7 @@
 #    READ_NODEDEFS: 8 (disable get_node_def call client-side)
 #    LOOKUP_NODES_LIMIT: 16 (limits get_node call client-side to
 #    csm_restriction_noderange)
+#    READ_PLAYERINFO: 32 (disable get_player_names call client-side)
 #    type: int
 # csm_restriction_flags = 30
```

Comment:
All feature restrictions should be disabled by default, that's what we decided earlier.